meteor/client/ui/Settings/SettingsMenu.tsx

```diff
@@ -293,9 +319,22 @@ function studioHasError(studio: Studio): boolean {
 
 interface SettingsMenuShowStyleProps {
 	showStyleBase: ShowStyleBase
+	showStyleVariantsForBase: ShowStyleVariant[]
+	blueprintsForBase?: Blueprint
 }
-function SettingsMenuShowStyle({ showStyleBase }: SettingsMenuShowStyleProps) {
+function SettingsMenuShowStyle({
+	showStyleBase,
+	showStyleVariantsForBase,
+	blueprintsForBase,
+}: SettingsMenuShowStyleProps) {
 	const { t } = useTranslation()
+	const variantConfigurationVerifier = new ShowStyleVariantConfigurationVerifier()
+	const errorInVariantBlueprintConfiguration =
```

Comment:
I expect this variable to hold an error, but it contains a boolean. It can be renamed to `hasErrorInVariantBlueprintConfiguration`.

Reply:
Updated